### PR TITLE
WMS service catalog data

### PR DESCRIPTION
**Pull request by williambohrmann3**
# Description

- Updates the catalog data to an online forecast service
- UI workarounds for long titles
- Update comments
- New screenshots (Guam forecast)

## Type of change

- Bug fix

## Platforms tested on

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI WinUI
- [x] MAUI Android
- [x] MAUI iOS
- [x] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] Code is commented and follows .NET conventions and standards
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] No unrelated changes have been made to any other code or project files
- [x] Screenshots are correct size and display in description tab (800 x 600 on Windows, MAUI mobile platforms use the MAUI Windows screenshot)

### CODE REVIEW

File: src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.jpg
Comment (duffh) [inline src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.jpg:1]: Minor UI issue here, the label is appearing behind the `ScrollViewer`. It is visible when you swipe across. 

image

File: src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
                 <TextBlock Grid.Row="0" Text="Select a WMS layer from the list for display:" />
-                <!--  This is a workaround until a proper TreeView is a available for all supported UWP platforms.  -->
-                <ListBox x:Name="MyDisplayList"
-                         Grid.Row="1"
-                         SelectionChanged="ListSelectionChanged"
-                         SelectionMode="Single" />
+                <ScrollViewer HorizontalScrollBarVisibility="Visible" HorizontalScrollMode="Enabled">
+                    <ListBox x:Name="MyDisplayList"
+                             Grid.Row="1"
+                             SelectionChanged="ListSelectionChanged"
```
Comment (duffh) [inline src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml:17]: The `Grid.Row` value is set on the wrong UI element.

Fix UI issue with: 

```
<ScrollViewer Grid.Row="1"
              HorizontalScrollBarVisibility="Visible"
              HorizontalScrollMode="Enabled">
    <ListBox x:Name="MyDisplayList"
             SelectionChanged="ListSelectionChanged"
             SelectionMode="Single" />
</ScrollViewer>
```

File: src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml.cs
Comment (duffh) [inline src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml.cs:1]: Missing full stop at the end of the comment on line 77.

"Updates the map with the latest layer selection."

File: src/WPF/WPF.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.jpg
Comment (duffh) [inline src/WPF/WPF.Viewer/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.jpg:1]:
```
<Border Width="400" Style="{StaticResource BorderStyle}">
```

The above change removes the scroll bar when the sample first loads. It's not a big issue but it looks slightly nicer.

File: src/MAUI/Maui.Samples/Samples/Layers/WmsServiceCatalog/wmsservicecatalog.jpg
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/Layers/WmsServiceCatalog/wmsservicecatalog.jpg:1]: There is something going on with the display of the `ListView`.

You might need to change the layout or width as the text wrapping looks odd.

**Review by duffh — CHANGES_REQUESTED** [review]
Looks good, just a few comments.

File: src/MAUI/Maui.Samples/Samples/Layers/WmsServiceCatalog/wmsservicecatalog.jpg
Comment (duffh) [reply src/MAUI/Maui.Samples/Samples/Layers/WmsServiceCatalog/wmsservicecatalog.jpg:1]: Increasing the width of the border didn't help it just made it look different but still not right. 

image

Comment (williambohrmann3) [reply src/MAUI/Maui.Samples/Samples/Layers/WmsServiceCatalog/wmsservicecatalog.jpg:1]: Added platform-specific XAML to handle this.

**Review by duffh — APPROVED** [review]

**Pull request merged**